### PR TITLE
Add Google Sign In

### PR DESCRIPTION
**Pull request by Jeffkungu**
#### What does this PR do?
Allows users to sign in using their Google accounts
#### Description of Task to be completed?
Add google Login to the app
#### How should this be manually tested?
After cloning the repo:
- Run the app in android studio
- Try Login in using Gmail

### CODE REVIEW

**Comment by phillipahereza** [comment]
@Jeffkungu We can't have 3 different activities for logging in, all login has to be in one activity. There's no way around that

**Comment by Jeffkungu** [comment]
I had a feeling about that, I had to get a confirmation thats why I sent the PR as early.

**Comment by phillipahereza** [comment]
@Jeffkungu I don't think there's any app in the world that has Facebook and Google login on different activities 😕

**Comment by Jeffkungu** [comment]
@phillipahereza Thank you, I'm still learning this basic conventions. I was trying out something and I needed to get your thoughts on it. I will send a better implementation of the same by end of day.

File: app/src/main/java/org/codeforafrica/citizenreporterandroid/ui/auth/LoginActivity.java [hunk 1]
```diff
@@ -61,6 +87,19 @@
     preferences = PreferenceManager.getDefaultSharedPreferences(this);
     editor = preferences.edit();
 
+    google_user_profile = (LinearLayout)findViewById(R.id.google_user_profile);
+    GoogleSignout = (Button)findViewById(R.id.google_sign_out);
+    GoogleSignin = (SignInButton)findViewById(R.id.google_sign_in);
+    Name = (TextView)findViewById(R.id.google_user_name);
```
Comment (phillipahereza) [inline app/src/main/java/org/codeforafrica/citizenreporterandroid/ui/auth/LoginActivity.java]: Variable names should *ALWAYS* start with lowercase letter

**Review by phillipahereza — CHANGES_REQUESTED** [review]
Basics Jeff, Basics!! 😕

**Comment by Jeffkungu** [comment]
Sorry about that, I have rectified it. won't happen again.

**Review by phillipahereza — APPROVED** [review]

**Pull request merged**